### ISSUE TITLE
Any plans to cut a new release?

### ISSUE THREAD

**Issue by adsr** [libinjection/libinjection#50]
I see there's been a bunch of activity since the last release.

**Comment by fzipi** [comment]
Probably. We still need to do stuff.

**Comment by fzipi** [comment]
I'm trying to 😅 . I'm trying to get approvals from people, but no one is taking a look yet.

**Comment by jirutka** [comment]
The last release is 8 years old, so it’s about time to finally make a release.

**Comment by fzipi** [comment]
Sure, do you want to help?

**Comment by fzipi** [comment]
Are you able to send PRs?

**Comment by jirutka** [comment]
Yes. What has changed since the last release, are there any backward incompatible changes or just additions?

**Comment by fzipi** [comment]
There has been many changes, and some pending PRs might make things incompatible. E.g. #65.

I would like to see for a new release, at least:
- #65 
- #57 
- others?

If you want to help, comment on the PRs, and try to add changes and comments to push them to the finish line.

**Comment by jirutka** [comment]
It would be better to release it now, and then finish the pending PRs, especially if they break compatibility. Some projects, e.g. [naxsi](https://github.com/wargio/naxsi), already depends on an arbitrary unreleased revision. I’m packaging this fork of the naxsi module and this situation with libinjection complicates things, because I would have to package some arbitrary revision instead of proper point release.

**Comment by fzipi** [comment]
Well, personally it will take some time, but I'll try to cut a new version this quarter. 🙏

**Comment by wargio** [comment]
That would be amazing if this happen so i can finally have a tagged version in naxsi!

**Comment by fzipi** [comment]
v4.0.0 (breaking) has been released. Have fun!
